testhelpers/core.go

```diff
@@ -184,3 +187,158 @@ func RequireCreateStorages(ctx context.Context, t *testing.T) (state.Tree, vm.St
 
 	return st, vms
 }
+
+type testStorage struct {
+	state interface{}
+}
+
+var _ exec.Storage = testStorage{}
+
+func (ts testStorage) Put(v interface{}) (cid.Cid, error) {
+	return cid.Cid{}, nil
+
+}
+func (ts testStorage) Get(cid.Cid) ([]byte, error) {
+	node, err := cbor.WrapObject(ts.state, types.DefaultHashFunction, -1)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	return node.RawData(), nil
+}
+
+func (ts testStorage) Commit(cid.Cid, cid.Cid) error {
+	return nil
+}
+
+func (ts testStorage) Head() cid.Cid {
+	return cid.Cid{}
+}
+
+// FakeVMContext creates the scaffold for faking out the vm context for direct calls to actors
+type FakeVMContext struct {
+	TestMessage        *types.Message
```

Comment:
`Test` prefix on these names isn't adding any information.

Reply:
The Prefix avoids a name collision with the accessor. I want to keep the relationship clear. I changed the "Test" prefix to a "Value" suffix, which at least hints at the relationship and the point of the suffix.